player/client.c

```diff
@@ -1726,29 +1726,47 @@ int mpv_opengl_cb_uninit_gl(mpv_opengl_cb_context *ctx)
 {
     return MPV_ERROR_NOT_IMPLEMENTED;
 }
+void mp_client_set_control_callback(struct mpv_opengl_cb_context *ctx,
+                                        mpv_opengl_cb_control_fn callback,
+                                        void *callback_ctx)
+{
+}
+void mp_client_set_icc_profile(struct mpv_opengl_cb_context *ctx, bstr icc_data)
+{
+}
+void mp_client_set_ambient_lux(struct mpv_opengl_cb_context *ctx, int lux)
+{
+}
 #endif
 
 int mpv_opengl_cb_render(mpv_opengl_cb_context *ctx, int fbo, int vp[4])
 {
     return mpv_opengl_cb_draw(ctx, fbo, vp[2], vp[3]);
 }
 
-void *mpv_get_sub_api(mpv_handle *ctx, mpv_sub_api sub_api)
+void *mpv_get_sub_api2(mpv_handle *ctx, mpv_sub_api sub_api, bool lock)
```

Comment:
It doesn't make sense to export this as public API. I think this is not actually exported (would require mpv.def entry), but still the function should not have a `mpv_` prefix to make clear it's private.

Reply:
changed that.